ES/ESmy023.xml

```diff
@@ -32,99 +32,377 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                                    <note>Accodring to the colophon, the writing of <ref target="#ms_i1"/> and
+                                                    <ref target="#ms_i2"/> was completed on <date>2 Ṗāgumen 1962 EC</date>
+                                                        (= <date>6 September 1969 CE</date>), by <persName ref="PRS13233DerarWaldaQirqos" role="scribe">ʾalaqā Dǝrār Walda Qirqos</persName>,
+                                                      at <placeName ref="LOC7348Legat"/>. The commissioner was
+                                                      <persName ref="PRS13234MehrataWaldaKidan" role="donor">wayzāro Məḥratā Walda Kidān</persName>.</note>
+                                                      <locus target="#89ra"/>
+                                                      <hi rend="rubric">ተፈጸመ፡ ዝንቱ፡ መጽሐፍ፡ አመ፡</hi> <date calendar="ethiopian" when-custom="1962" when="1969">፪ለጳጉሜን፡ በ፲ወ፱፻፷ወ፪ዓመተ፡</date>
+                                                       ምሕረት፡ በ፴ወ፱ዘመነ፡ መንግሥቱ፡ ለንጉሥነ፡ ኃይለ፡ <hi rend="rubric">ሥላሴ፡</hi> በእደ፡ ጸሐፊሁ፡ አለቃ፡ ድራር፡ ወልደ፡ ቂርቆስ፡ ዘብሔረ፡
+                                                         ልጋት፡ ወዘአጽሐፈቶ፡ ወይዘሮ፡ ምሕረታ፡ ወልደ፡ ኪዳን፡ በ፻ወ፳ብሩር፡ ወወሀበቶ፡ ለታቦተ፡ ዮሐንሰ፡ ደብረ፡ ማዕጾ፨ ዘሠረቆ፡
+                                                         ወዘፈሐቆ፡ ርጉመ፡ ይኩን፡ ከመ፡ አርዮስ፡ በሥልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ለይኩን፡ ለይኩን፡ ለይኩን፨
+
+                                                         <foreign xml:lang="en">This book was completed on
+                                                           <date>2 Ṗagumen, in the year of mercy 1962</date>, in the 39th year of the reign of our king <persName ref="PRS5226haylase">Ḫāyla Śǝllāse</persName>, by
+                                                           the hands of the scribe <persName ref="PRS13233DerarWaldaQirqos" role="scribe">ʾalaqā Dǝrār Walda Qirqos</persName>
+                                                            of the country of <placeName ref="LOC7348Legat">Lǝgāt</placeName>. And
+                                                         the one who caused him to write is <persName ref="PRS13234MehrataWaldaKidan" role="donor">wayzāro Məḥratā Walda Kidān</persName>,
+                                                         for 120 <foreign xml:lang="gez">Bǝrur</foreign>, and
+                                                             she gave it to the <foreign xml:lang="gez">tābot</foreign> of <placeName ref="LOC2320DabraM">Yoḥannǝs of Dabra Māʿṣo</placeName>. Anyone who stole it, or
+                                                             who erased it, be cursed like Arius by authority of Peter and Paul. So be it,
+                                                           so be it, so be it.</foreign></colophon>
+                                                             </msItem>
+                                                          </msItem>
+
+                          <msItem xml:id="ms_i3">
```

Comment:
since this comes after the colophon - are we sure this is part of the original composition and not an addition?

Reply:
We cannot be sure. This might have been added later or just after the colophon, but it is the "original programm" logically extended by the main scribe. I would not consider it an addition.